cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java

```diff
@@ -38,6 +38,9 @@
 import java.util.Date;
```

Comment:
CNR entry corresponding to this PR should be added.

Reply:
Fixed in [155a1ab](https://github.com/telefonicaid/fiware-cygnus/pull/1934/commits/155a1ab4d2f52093e588f85d399ec32690ad4402)